packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -734,6 +735,11 @@ export class BidiPage extends Page {
         !String.prototype.startsWith.call(cookieUrl || '', 'data:'),
         `Data URL page can not have cookie "${cookie.name}"`,
       );
+      assert(
```

Comment:
let's add a TODO comment in these assertions linking to https://github.com/GoogleChromeLabs/chromium-bidi/issues/2434

Reply:
Done